dev/com.ibm.ws.cdi.extension_fat/publish/servers/weldDevModeServer/.gitignore

```diff
@@ -0,0 +1 @@
+/dropins
```

Comment:
Why is this needed? We shouldn't be putting anything into `/dropins` at build time.